### PR TITLE
[docker backend] - Add config flag to set refreshSeconds for swarmmode ticker

### PR DESCRIPTION
**Pull request by WTFKr0**
### What does this PR do?

This PR allow user to set a custom refresh interval for polling swarm mode socket for docker backend
The default value is 15
The new flag to set in config is `--docker.refreshSeconds`

### CODE REVIEW

File: provider/docker/docker.go [hunk 1]
```diff
@@ -46,6 +44,7 @@ type Provider struct {
 	UseBindPortIP         bool             `description:"Use the ip address from the bound port, rather than from the inner network" export:"true"`
 	SwarmMode             bool             `description:"Use Docker on Swarm Mode" export:"true"`
 	Network               string           `description:"Default Docker network used" export:"true"`
+	RefreshSeconds        int              `description:"Polling interval for swarm mode (in seconds)" export:"true"`
```
Comment (nmengin) [inline provider/docker/docker.go]: WDYT to rename the option into `SwarmRefreshPeriod`?

File: cmd/configuration.go [hunk 2]
```diff
@@ -46,6 +46,7 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 	defaultDocker.SwarmMode = false
+	defaultDocker.SwarmRefreshPeriod = 15
```
Comment (nmengin) [inline cmd/configuration.go]: Sorry, but I made a mistake, I would say `SwarmModeRefreshPeriod` in the way to be homogeneouse with the option `SwarmMode` and avoid misunderstood with the old Swarm orchestrator...

**Comment by ldez** [comment]
Could you rebase on v1.7?

**Comment by WTFKr0** [comment]
Thanx for taking this !

**Pull request merged**